README.md

```diff
@@ -173,7 +163,7 @@ You need valid developer credentials for `cmsFronts` and `workflow`.
 You can get keys temporary keys from `janus`. Note that the recommended way to get Janus credentials for *facia-tool* is to use feria and run:
 
 ```
-$ feria cmsFronts && feria --access s3-read workflow && feria --access sqs-consumer frontend
```

Comment:
I've added it back, but I'll add a note about the frontend credentials after the tool is no longer writing on the frontend bucket.